Source/OpalGraphics/cairo/CairoFontX11.m

```diff
@@ -163,22 +169,23 @@ - (CFStringRef) copyFullName;
         result = [[NSString alloc] initWithBytes: nameStruct.string length: nameStruct.string_len encoding: NSUTF16BigEndianStringEncoding];
       }
       else if (nameStruct.platform_id == TT_PLATFORM_MACINTOSH &&
-               nameStruct.encoding_id == TT_MAC_ID_ROMAN)
+        nameStruct.encoding_id == TT_MAC_ID_ROMAN)
       {
         result = [[NSString alloc] initWithBytes: nameStruct.string length: nameStruct.string_len encoding: NSMacOSRomanStringEncoding];
       }
       else if (nameStruct.platform_id == TT_PLATFORM_MICROSOFT &&
-               nameStruct.encoding_id == TT_MS_ID_UNICODE_CS)
+        nameStruct.encoding_id == TT_MS_ID_UNICODE_CS)

```

Comment:
I believe this fix is wrong, and the original indentation should be correct. I don't have GNU style guide at hand, so I may be wrong.

There are multiple instances of this.